### PR TITLE
fix UI/Layout of explore page for big screens

### PR DESCRIPTION
**Pull request by oussamaBGZ**
## Description
Fix the layout of the explore page for big screen 

## Related Issue
#686 

## Screenshot:
image

### CODE REVIEW

**Comment by PickleNik** [comment]
the reason this wasn't done from the very beginning is to avoid this cropping
Screenshot 2023-09-19 at 10 06 35 AM
Screenshot 2023-09-19 at 10 07 16 AM
the right way to fix the issue would be to add padding to the left side of the carousel that is equal to half the difference between container max-width and screen width

**Comment by PickleNik** [comment]
an alternative for VERY large screens would be to make it fade-out after a certain point with an overlay absolutely positioned within the carousel container with 
`pointer-events-none bg-gradient-to-r from-background via-transparent to-background via-10% to-90%`
to achieve something similar to this:

https://github.com/typehero/typehero/assets/31113245/356e7914-94a3-4c85-be62-7021c0bbcb67

instead of it just cropping off in the middle of the screen which doesn't look nice imo

**Comment by oussamaBGZ** [comment]
demo let me know:

https://github.com/typehero/typehero/assets/30552331/5c8ddac3-6304-40db-ab7c-6110a0a65556

**Comment by PickleNik** [comment]
I don't think we should copy LeetCode's stretching of the page to the entire screen width

Screenshot 2023-09-20 at 8 08 21 PM

**Comment by PickleNik** [comment]
I feel like with the new fade out on the sides, if we kept text/buttons & all the other stuff bound to container width like it was before, but let carousels be full-width that would look better
Screenshot 2023-09-20 at 8 12 20 PM

I'm still not sure about the purpose changing the `xl` bound to `1400px`

**Comment by PickleNik** [comment]
something like this maybe?

https://github.com/typehero/typehero/assets/31113245/933475a7-b58d-47c6-babe-3ef096dafa91

**Comment by PickleNik** [comment]
or maybe even simpler without calculating the padding, just slap `justify-center` on the carousel (the lack of beginner challanges makes it look weird)
https://github.com/typehero/typehero/assets/31113245/a7658998-f04e-449e-b98c-4b158baa13d2

**Comment by oussamaBGZ** [comment]
> something like this maybe?
> 
>  Screen.Recording.2023-09-20.at.8.21.12.PM.mov

Ahh ok

**Comment by bautistaaa** [comment]
> or maybe even simpler without calculating the padding, just slap `justify-center` on the carousel (the lack of beginner challanges makes it look weird)
> 
>  Screen.Recording.2023-09-20.at.8.23.17.PM.mov

we'll always have full carousels (or should)

**Comment by PickleNik** [comment]
> we'll always have full carousels (or should)

in that case left-side padding would work better i think

**Comment by bautistaaa** [comment]
> > we'll always have full carousels (or should)
> 
> in that case left-side padding would work better i think

only place i think it would not be full is "your tracks". i think that might be reusing something. not sure

**Comment by oussamaBGZ** [comment]
https://github.com/typehero/typehero/assets/30552331/f8fb7c7b-8c1d-4be9-8f93-daea6ed24696

File: apps/web/tailwind.config.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ module.exports = {
       screens: {
         // iPad Pro vertical is 1024px exactly
         lg: '1025px',
+        xl: '1400px',
```
[the remark this answers is not in the document]
Comment (PickleNik) [reply apps/web/tailwind.config.ts:18]: @oussamaBGZ

**Review by PickleNik — APPROVED** [review]
gltm

File: apps/web/tailwind.config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (oussamaBGZ) [reply apps/web/tailwind.config.ts:18]: no reason for it i will remove it

**Comment by bautistaaa** [comment]
very nice

**Pull request merged**